### PR TITLE
feat: add monolithic docker

### PR DESCRIPTION
**Pull request by Gaitholabi**

### CODE REVIEW

File: templates/docker/uninstall.sh [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+printf "\033[0;33m|       WARNING       |\033[0m\n"
+printf "\033[0;33m|---------------------|\033[0m\n"
+echo
+printf "\033[0;33mThis action will irreversibly uninstall your help desk. This includes, but not limited to, permanent removal of the database and all user files.\033[0m\n"
+printf "\033[0;33mDo NOT continue without a recent full system backup. You can take a backup of your system by executing: make backup\033[0m\n"
+echo
+printf "\033[0;33mPress CTRL + C to cancel this operation. Waiting 15 seconds...\033[0m\n"
+sleep 15
+
+
```
Comment (bytestream) [inline templates/docker/uninstall.sh:13]:
```suggestion
```

File: templates/docker/start.sh [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+--volume supportpal_redis:/redis-data \
+--volume supportpal_storage:/var/www/supportpal/storage \
+--volume "${PWD}/customization/:/customization" \
+--volume "${PWD}/php/custom.ini:/${CLI_PATH}" \
+--volume "${PWD}/php/custom.ini:/${FPM_PATH}" \
+-p "127.0.0.1:${WEB_PORT}:80" \
+-p "127.0.0.1:${WS_PORT}:6001" \
+--name supportpal \
+-d \
+--rm -t -i "supportpal:${VERSION}" /sbin/my_init > /dev/null
```
Comment (bytestream) [inline templates/docker/start.sh:47]: I think ideally we want to start publishing a latest image, then all they need to do is `docker pull` and happy days? No modifying files, pulling git repos etc.

File: templates/docker/start.sh [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+docker run \
+--volume supportpal_db:/var/lib/mysql \
+--volume supportpal_config:/var/www/supportpal/config/production \
+--volume supportpal_redis:/redis-data \
+--volume supportpal_storage:/var/www/supportpal/storage \
+--volume "${PWD}/customization/:/customization" \
+--volume "${PWD}/php/custom.ini:/${CLI_PATH}" \
+--volume "${PWD}/php/custom.ini:/${FPM_PATH}" \
+-p "127.0.0.1:${WEB_PORT}:80" \
+-p "127.0.0.1:${WS_PORT}:6001" \
```
Comment (bytestream) [inline templates/docker/start.sh:44]: The web socket server shouldn't be exposed publicly? Easier to configure if it stays unencrypted, and traffic goes via the web server.

File: templates/docker/setup.sh [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+  username="$(docker run --rm public.ecr.aws/supportpal/helpdesk-configurator:latest openssl rand -base64 14)"
+  user_password="$(docker run --rm public.ecr.aws/supportpal/helpdesk-configurator:latest openssl rand -base64 14)"
+
+  docker run \
+  --volume supportpal_db:/var/lib/mysql \
+  -e MYSQL_DATABASE="${database}" \
+  -e MYSQL_ROOT_PASSWORD="${root_password}" \
+  -e MYSQL_USER="${username}" \
+  -e MYSQL_PASSWORD="${user_password}" \
+  --rm sp-mysql > /dev/null;
```
Comment (bytestream) [inline templates/docker/setup.sh:86]: Might be able to move this to the `docker-entrypoint.sh` so it's one less step?

**Comment by Gaitholabi** [comment]
Closing in favour of running a single `docker run ---` command.

**Pull request closed without merging**